### PR TITLE
remove info-at-msdc

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
while replacing the email address with sales-approved text, cleaned up this topic.

### CODE REVIEW

**Comment by Mulesoft-Mysti** [comment]
If you don't like the links at the top, I'm happy to take them out. Just thought it would be interesting to see how they look.

**Comment by Mulesoft-Mysti** [comment]
When we fix the bug with headings not appearing in ToC, these long titles will not look great. Should I change them all to <b> and not ==? If I do that, we lose the links at the top...

File: modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc [hunk 1]
```diff
@@ -2,67 +2,87 @@
...
+=== What can I do if my trial is about to expire?
 
-Check that you are using the correct URL to log in to your organization. If your organization uses an external identity management system, your login URL uses this pattern: *anypoint.mulesoft.com/accounts/login/yourorganizationname*
+Talk to your account representative or use https://www.mulesoft.com/contact[the MuleSoft contact form] to request an Enterprise license for your on-premises Mule Runtime instances or to create a subscription account on CloudHub.
 
-If your organization is not using an external identity management system and your login credentials are not working, reset your password or consult your Organization Administrator to ensure that your account is still active.
+=== Where do I log in with external identity management?
 
-== I accepted an invitation and logged in, but I cannot see anything in the platform.
+If your organization uses external identity management, log into account at *anypoint.mulesoft.com/accounts/login/_yourorganizationname_*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc]: Line 37--I think "your" is missing in "log into...account"

File: modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc [hunk 2]
```diff
@@ -2,67 +2,87 @@
...
+[NOTE]
+The *Permissions* tab doesn't display any users with permissions based on a custom role, because roles can't be managed from API Version Details pages. Contact your Organization Administrator for information about role-based permissions to your API version.
 
-Log out of your account in order to browse Developer Portals for other organizations. While you are logged in to your own organization, you can only access your own organization's Developer Portal. Note that if you wish to request access to an API of another organization, you need to have an Anypoint Platform account for that organization.
+=== How do I browse Developer Portals?
 
-== I have found an API that I would like to use, but I'm not a member of the organization that owns the API. How can I request access to this API?
+Log out of your account in order to browse Developer Portals for other organizations. While you are logged in to your own organization, you can only access your own organization's Developer Portal.
```
Comment (valkyrie69) [inline modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc]: Line 79, "in order" is extraneous (IMO)

File: modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc [hunk 3]
```diff
@@ -2,67 +2,87 @@
...
+[NOTE]
+The *Permissions* tab doesn't display any users with permissions based on a custom role, because roles can't be managed from API Version Details pages. Contact your Organization Administrator for information about role-based permissions to your API version.
 
-Log out of your account in order to browse Developer Portals for other organizations. While you are logged in to your own organization, you can only access your own organization's Developer Portal. Note that if you wish to request access to an API of another organization, you need to have an Anypoint Platform account for that organization.
+=== How do I browse Developer Portals?
 
-== I have found an API that I would like to use, but I'm not a member of the organization that owns the API. How can I request access to this API?
+Log out of your account in order to browse Developer Portals for other organizations. While you are logged in to your own organization, you can only access your own organization's Developer Portal. 
+[NOTE]
+If you wish to request access to an API of another organization, you need to have an Anypoint Platform account for that organization.
```
Comment (valkyrie69) [inline modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc]: Just "To request access to an API..."

?

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**